src/screens/WhotoFollow/elements/Label.js

```diff
@@ -48,6 +48,6 @@ const styles = StyleSheet.create({
     fontWeight: 'bold',
     fontSize: normalizeFontSize(12),
     lineHeight: normalizeFontSize(18),
-    color: COLORS.black000
+    color: COLORS.white2
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the text has been changed from black to white. Please ensure that this change doesn't affect readability on the background it's being used, especially if the background is light-colored.
```diff
-    color: COLORS.black000
+    color: COLORS.white2
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->